### DISCUSSION TITLE
Anyone of core team wants to talk sqlc in go podcast()?

### DISCUSSION THREAD

**Discussion by dstpierre** [sqlc-dev/sqlc#3469] [Issue Triage]
Hi,

I was wondering if anyone of the core maintainers / contributors would like to join me in my podcast on Go for an episode on sqlc?

I believe it would be a great episode, `sqlc` seems to get more and more traction, there's a lot we could talk about.

I'm not sure it's the right place for such an invitation, but I'd be happy to talk about it, as I'm using it as well and I'm a fan.

It could be 1 to 4 guests. Let me know if anyone's interested.

Dominic

**Comment by kyleconroy** [comment]
Appreciate the invite, but sadly don't have time right now. Happy to answer questions over email if you'd still like to discuss sqlc.
